v1/src/main/java/com/google/cloud/teleport/spanner/AvroUtil.java

```diff
@@ -68,4 +70,33 @@ public static Schema unpackNullable(Schema schema) {
     }
     return null;
   }
+
+  /**
+   * For named schema, the database object name could have dot(.). When building a avro schema
+   * object, a name with dot(.) is special. See org.apache.avro.Schema.Name#name for more
+   * detail. So we use avro property to store the database object name in newer version, this
+   * function thus provides a way to get name in a compatible way.
```

Comment:
Can you add a couple of examples here? Like what would be returned with schema.getProp(SPANNER_NAME) and schema.getName()